### PR TITLE
Limit title length to 256 characters

### LINKED ISSUE #110: Prevent project titles with 1,000,000,000+ characters
see: https://github.com/mozilla/webmaker-android/issues/2394

### PR DESCRIPTION
**Pull request by cadecairos**
Fixes #110

### CODE REVIEW

**Comment by cadecairos** [comment]
need to find a reviewer....

File: services/api/routes/authenticated.js [hunk 1]
```diff
@@ -139,7 +139,7 @@ var routes = [
           user: numericSchema
         },
         payload: {
-          title: Joi.string().required(),
+          title: Joi.string().max(250).required(),
```
Comment (ashleygwilliams) [inline services/api/routes/authenticated.js]: did you mean 256? (referencing the limit in the title..)

Comment (cadecairos) [inline services/api/routes/authenticated.js]: hahaha yes

Comment (ashleygwilliams) [inline services/api/routes/authenticated.js]: :mag_right: :laughing:

**Comment by ashleygwilliams** [comment]
once that number is changed i'm r+ on this.

**Comment by ashleygwilliams** [comment]
r+

**Pull request merged**